x-pack/plugins/security_solution/public/resolver/view/panels/copyable_panel_field.tsx

```diff
@@ -34,6 +34,12 @@ const StyledCopyableField = styled.div<StyledCopyableField>`
   }
 `;
 
+const TRANSLATED_COPY_TO_CLIPBOARD = i18n.translate(
```

Comment:
can you just put this inline since its a string, or otherwise use a camel case variable name.